inscriptions/ISic3098.xml

```diff
@@ -179,28 +184,27 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331423">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — — — — — — —</supplied>
-                    <lb n="2"/>
-                    <unclear>Γ</unclear>ΗΙΙ
-                    <unclear>ΑΟ</unclear>
-                    <supplied reason="lost">— — — — ὑπατίᾳ</supplied>
-                    <lb n="3"/>τοῦ δεσπ<supplied reason="lost">ότου ἡμῶν Θεο</supplied>-<lb n="4"/>δοσίου
-                    <supplied reason="lost">τὸ —ʹ κ(αὶ) — —</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><orig><unclear>Γ</unclear>ΗΙΙ<unclear>ΑΟ</unclear></orig><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">ὑπατίᾳ</supplied>
+                    <lb n="3"/>τοῦ δεσπ<supplied reason="lost">ότου</supplied> <supplied reason="lost">ἡμῶν</supplied> <supplied reason="lost">Θεο</supplied>
+                    <lb n="4" break="no"/>δοσίου <supplied reason="lost">τὸ</supplied> <supplied reason="lost"><num value="?">?</num></supplied> <supplied reason="lost"><expan><abbr>κ</abbr><ex>αὶ</ex></expan></supplied> <!-- do we want to flag here that there is a number even if we don't know its value? -->
```

Comment:
That supplied number - does it have a supposed value or range of values, or do we have no clue? What about extent of the gap itself - any guesses as to how many character we're missing? Either way, it should look something like this: <num><supplied reason="lost" extent="unknown" unit="character"/></num>. The `@value` only takes numerical content, so if we don't have a value we don't put a '?' there.